packages/cli/src/util/logger.js

```diff
@@ -23,13 +23,32 @@ const error = (...messages: Array<string>) => {
   );
 };
 
+const logError = (err: Error) => {
+  console.error(`${chalk.black.bgRed(' ERROR ')} ${chalk.red(err.message)} \n`);
+  if (err instanceof ProcessError) {
+    console.error(`${chalk.grey(err.processError)}`);
+  }
+};
+
 const debug = (...messages: Array<string>) => {
   console.log(`${chalk.black.bgWhite(' DEBUG ')} ${joinMessages(messages)}`);
 };
 
+class ProcessError extends Error {
+  constructor(msg: string, processError: string) {
+    super(msg);
+    this.processError = processError;
+    Error.captureStackTrace(this, ProcessError);
+  }
+
+  processError: string;
+}
+
 module.exports = {
   info,
   warn,
   error,
   debug,
+  logError,
```

Comment:
maybe `logProcessError`? It's very scoped to `ProcessError`

Reply:
It is scoped for now, but the intent is to log all other kinds of errors too. We use it in the main cli interface where we catch the error from the command and don't know it's type yet.

Reply:
See: https://github.com/react-native-community/react-native-cli/pull/139/files#diff-97059d274ee5d7bbcc5730ddee2a98daR30